.docker/wp-cli.yml

```diff
@@ -0,0 +1,2 @@
+path: /usr/src/public_html/mu
+url: wordcamp.test
```

Comment:
```suggestion
```

`MIGHT` - We can't use that on sandboxes/production, so it might be best to just leave it out. Alternatively, we could `gitignore` this file? But it would be nice to have one on production too, so maybe the script that pushes to the SVN repo could swap `.test` for `.org` or something?

Reply:
Its added to `public_html` during Docker compose, but otherwise its always inside `.docker` folder. So it should not cause an issue on sandboxes/production. What do you think?

Reply:
👍